tests/node-tests.js

```diff
@@ -1,64 +1,61 @@
-(function() {
-	"use strict";
```

Comment:
I removed the IIFE wrapper. Didn't seem necessary in a node-only context and it caused eslint to flag the use of sync node APIs at the "top level" (since IIFE is not the top level).